package.json

```diff
@@ -98,9 +105,9 @@
     "react-blockies": "^1.4.0",
     "react-display-name": "^0.2.3",
     "react-onclickout": "^2.0.8",
-    "react-spring": "^7.2.11",
+    "react-spring": "7.2.10",
```

Comment:
The version 7.2.11 not exist anymore, so was neeed rollback to resolve some conflict on types definition.